app/code/Magento/Captcha/view/frontend/layout/default.xml

```diff
@@ -7,6 +7,19 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
+        <referenceBlock name="contactForm">
```

Comment:
Why it's needed? You can't reference block that have not been created. 
cacheable="false" will disable FPC for all default layouts.